<launch>
    <arg name="map_name" default="4way"/>
    <arg name="n_duckies" default="3"/>
    <arg name="rviz_config" default="$(find flock_simulator)/config/$(arg map_name).rviz"/>

    <node pkg="flock_simulator" name="flock_simulator_node"
    type="flock_simulator_node.py" output="screen">
        <param name="map_name" value="$(arg map_name)"/>
        <param name="n_duckies" value="$(arg n_duckies)"/>
    </node>

    <node pkg="flock_planner" name="flock_planner_node"
        type="flock_planner_node.py" output="screen" />

    <include file="$(find flock_simulator)/launch/visualization.launch">
        <arg name="map_name" value="$(arg map_name)"/>
        <arg name="rviz_config" value="$(arg rviz_config)"/>
    </include>
</launch>
